roles/installer/templates/ldap.py.j2

```diff
@@ -0,0 +1,6 @@
+AUTH_LDAP_GLOBAL_OPTIONS = {
+{% if ldap_cacert_secret %}
```

Comment:
nit: This would be  best set as `ldap_cacert_ca_crt` and not `ldap_cacert_secret`. Just in case `ldap_cacert_secret` is actually specified but no `ldap-ca.crt` is available inside.

Reply:
Yes, it does make sense. I´ll update the PR @Spredzy++